src/script/ismine.h

```diff
@@ -28,6 +30,9 @@ enum isminetype
 /** used for bitflags of isminetype */
 typedef uint8_t isminefilter;
 
+// TODO support for multiwallet ?
+extern bool g_anyonecanspend_aremine;
```

Comment:
Not sure this needs multiwallet support. It's for testing only.

Reply:
I'll remove the comment, sorry.